clefos/content.md

```diff
@@ -0,0 +1,92 @@
+# ClefOS
+
+ClefOS Linux is a community-supported distribution for IBM Z (aka "mainframe") derived from sources freely provided to the public by [CentOS](http://vault.clefos.org/) which in turn is derived from the [Red Hat](ftp://ftp.redhat.com/pub/redhat/linux/enterprise/) sources for Red Hat Enterprise Linux (RHEL). As such, ClefOS Linux aims to be functionally compatible with CentOS and RHEL. The ClefOS Project mainly changes packages to remove upstream vendor branding and artwork. ClefOS Linux is no-cost and free to redistribute. Each ClefOS Linux version is maintained and released according to the CentOS schedule.
+
+%%LOGO%%
+
+## ClefOS image documentation
+
+The `clefos:latest` tag is always the most recent version currently available.
+
+### Building the Base Image
+
+The image is built via the `make` command which will create the tarball, build the image, and push the image to the repository.
+
+The `createBase.sh` script is used to create the tarball for the docker build command. The script uses the yum command with the `tsflags=nodocs` option set to reduce the size of the image. In addition a lot of the locale files are eliminated from the image.
+
+The `VERSION` file contains the id of the current ClefOS version and will be added as a label within the image.
+
+### Rolling builds
+
+The ClefOS Project offers regularly updated images for all active releases. These images will be updated monthly or as needed for emergency fixes. These rolling updates are tagged with the major version number and minor tags as well. For example, if 7.4.1708 is the most current then the build will result in `clefos:7` and `clefos:7.4.1708`. When the next minor level is available then `clefos:7` and `clefos:7.x.yymm` will be identical.
```

Comment:
Again, these should use `%%IMAGE%%:xxx` so that they update properly for https://hub.docker.com/r/s390x/clefos/.